### DISCUSSION TITLE
How to get the average value of a variable at a certain boundary?

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#23106] [Q&A General] [answered]
hello,
   I need to get average value of a variable at a certain boudary at every time step? Is there any postprocessor or auxkernel could achieve this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

there s a SideAverageValue postprocessor that does this
https://mooseframework.inl.gov/moose/source/postprocessors/SideAverageValue.html

Guillaume

**Reply by xiaojbing** [reply]
thank you, I will try
